src/components/join/JoinForm.tsx

```
@@ -0,0 +1,128 @@
+        if (!nicknameAvailable) {
+            setError('nickName', {
+                message: '닉네임 중복 확인을 해주세요!',
+            });
+            return;
+        }
+
+        try {
+            await signUp(data);
+            alert('회원가입이 완료되었습니다!');
+            router.push('/login');
+        } catch (err: any) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)